src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -1245,13 +1246,13 @@ static void add_stream_kafka_command(int connFd, std::string &kafka_server_IP, c
     bzero(topic_name, FRONTEND_DATA_LENGTH + 1);
     read(connFd, topic_name, FRONTEND_DATA_LENGTH);
 
-    string con_message = "Received the kafka topic";
-    int con_result_wr = write(connFd, con_message.c_str(), con_message.length());
-    if (con_result_wr < 0) {
-        frontend_logger.error("Error writing to socket");
-        *loop_exit_p = true;
-        return;
-    }
+//    string con_message = "Received the kafka topic";
```

Comment:
Why is this commented out? Can we remove the commented code segment?

Reply:
Uncommented. We need it.

Reply:
Ok